src/host/services/NMEAService.cpp

```diff
@@ -149,8 +150,9 @@ void NMEAService::loop() {
   DEBUG("Found %i sentences waiting", receiveQueue.size());
   for (LinkedList<NMEASentence>::iterator it = receiveQueue.begin(); it != receiveQueue.end(); it++) {
     if (it->isValid()) {
-      KBoxMetrics.countEvent(rxValidEvent);
-      this->sendMessage(*it);
+      KBoxMetrics.event(rxValidEvent);
+      // DEBUG( "Rx NMEAValidCounter: %i", KBoxMetrics.countEvent(rxValidEvent) );
```

Comment:
Can you remove this line? WE are not going to need it.

Reply:
debug message is deleted.
the readme.txt was made by platformio automatically (always restored when I deleted it)
I will have to find a way to experiment and try, including debug messages for testing without "damaging" your code....

Reply:
It looks like you did not commit the change to remove that debug message. Can you remove it?